components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -780,6 +784,19 @@ + (CGFloat)defaultHeight {
   return kSpinnerRadius * 2.f;
 }
 
++ (NSArray<UIColor *> *)defaultCycleColors {
+  static NSArray<UIColor *> *s_defaultCycleColors;
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    s_defaultCycleColors =
+    @[ [[UIColor alloc] initWithRed:0.129f green:0.588f blue:0.953f alpha:1],
```

Comment:
I believe that would be outside the scope of this particular issue, and I opened Issue #1546 to track the request.